### PR TITLE
Add option in server to enable regeneration vitals in combat

### PR DESCRIPTION
**Pull request by blinkuz**
It is a very basic PR but it is linked to this issue https://github.com/AscensionGameDev/Intersect-Engine/issues/506, I have used the suggestions that they had written but without eliminating the default behavior.

### CODE REVIEW

File: Intersect (Core)/Config/Options.cs [hunk 1]
```diff
@@ -128,6 +128,8 @@ public class Options
 
         public static int RegenTime => Instance.CombatOpts.RegenTime;
 
+        public static bool CombatRegen => Instance.CombatOpts.CombatRegen;
```
Comment (Cheshire92) [inline Intersect (Core)/Config/Options.cs:131]: We're not doing new top level options like this anymore.
Please refer to the option through the category it is in.

File: Intersect (Core)/Config/CombatOptions.cs [hunk 2]
```diff
@@ -17,6 +17,8 @@ public class CombatOptions
         //Combat
         public int RegenTime = 3000; //3 seconds
 
+        public bool CombatRegen = false; // Enable or disable regeneration in combat.
```
Comment (Cheshire92) [inline Intersect (Core)/Config/CombatOptions.cs:20]: Instead of making this a comment could you please make it actual XML documentation?

**Review by Cheshire92 — CHANGES_REQUESTED** [review]

File: Intersect (Core)/Config/CombatOptions.cs (hunk 2 above)
Comment (blinkuz) [reply Intersect (Core)/Config/CombatOptions.cs:20]: Last night I considered doing it but it was 1AM, today I will adjust the comment correctly.

File: Intersect (Core)/Config/Options.cs (hunk 1 above)
Comment (blinkuz) [reply Intersect (Core)/Config/Options.cs:131]: So would it be to remove the top-level option and then just call an:

` Options.Instance.CombatOpts.CombatRegen`

?

Comment (Cheshire92) [reply Intersect (Core)/Config/Options.cs:131]: Yeah, might be worth naming it something descriptive as well that tells the user what it does.

**Review by Cheshire92 — CHANGES_REQUESTED** [review]
Ah sorry missed this one between work. 

If you'd remove the two extra newlines you've added this will pretty much be good to go.

**Comment by blinkuz** [comment]
> Ah sorry missed this one between work.
> 
> If you'd remove the two extra newlines you've added this will pretty much be good to go.

But those lines already existed originally, I had previously deleted them and then added them to leave everything as it was or am I wrong?

**Review by Cheshire92 — APPROVED** [review]

**Pull request merged**